src/Decoder/RequestParamsDecoder.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Webgriffe\SyliusNexiPlugin\Decoder;
+
+final class RequestParamsDecoder implements RequestParamsDecoderInterface
+{
+    public function decode(array $requestParams): array
+    {
+        foreach (array_keys($requestParams) as $key) {
+            $requestParams[$key] = mb_convert_encoding($requestParams[$key], 'UTF-8', 'ISO-8859-1');
```

Comment:
Se l'input è in codifica ISO-8859-1 tutto bene, ma se non lo fosse?

Monolog, che ha lo stesso problema quando deve loggare in JSON, fa così: https://github.com/Seldaek/monolog/blob/60ad5183b5e5d6c9d4047e9f3072d36071dcc161/src/Monolog/Utils.php#L197-L218

Che sia il caso di fare la stessa cosa?